hooks/synths/useFeeReclaimPeriods.ts

```diff
@@ -0,0 +1,15 @@
+import { useMemo } from 'react';
+
+import useFeeReclaimPeriodsQuery from 'queries/synths/useFeeReclaimPeriodsQuery';
+import { SynthFeeAndWaitingPeriod } from 'queries/trades/types';
+
+const useFeeReclaimPeriods = (): SynthFeeAndWaitingPeriod[] => {
```

Comment:
same here. it can just be inside the component. doesn't have to be wrapped in another hook